sale_loyalty_multi_gift/tests/test_sale_loyalty_multi_gift.py

```diff
@@ -104,3 +104,60 @@ def test_04_test_sale_coupon_test_multi_gift(self):
         self.assertEqual(2, discount_line_product_2.product_uom_qty)
         self.assertEqual(3, discount_line_product_4.product_uom_qty)
         self.assertFalse(discount_line_product_3)
+
+    def test_delete_01_loyalty_multi_gift_ids(self):
```

Comment:
Put this test in your migration commit

Reply:
This commit is to remove it when the dependency is merged because it is not a test of this module.

Reply:
Which one is it then? :thinking:

Reply:
This is the test you used to debug the duplicate records error in loyalty_multi_gift.

Reply:
And why shouldn't we keep it as a regression test?

Reply:
But I think it would be more suitable to keep it in loyalty_multi_gift, don't you?

Reply:
Ok, while it test that case :)

Reply:
done